src/dispatch/plugins/dispatch_slack/workflow.py

```diff
@@ -243,6 +243,10 @@ def handle_workflow_submission_event(
         {
             "externalRef": f"{DISPATCH_UI_URL}/{instance.incident.project.organization.name}/incidents/{instance.incident.name}?project={instance.incident.project.name}",
             "workflowInstanceId": instance.id,
+            "incident_name": instance.incident.name,
```

Comment:
Having these metadata in payload is very handy, without making any extra api calls back to dispatch.